frontend/express/public/localization/dashboard/dashboard.properties

```diff
@@ -52,18 +52,19 @@ web.common.bar.top-resolution.description = Top 3 resolution settings of the dev
 common.bar.top-carrier = Top Carriers
 common.bar.top-users = TOP USERS
 common.bar.top-platform = Top Platforms
+common.bar.top-platform_ = TOP PLATFORMS
 common.bar.top-platform.description = Top 3 platforms of your users' sessions, in the selected time period.
 web.common.bar.top-platform.description = Top 3 platforms of your visiors' sessions, in the selected time period.
 common.bar.top-platform-version = Top Platform Versions
 common.bar.top-platform-version.description = Top 3 versions of the platforms of your users' sessions, in the selected time period.
 web.common.bar.top-platform-version.description = Top 3 versions of the platforms of your visitors' sessions, in the selected time period.
-common.bar.top-devices = Top  Devices
+common.bar.top-devices = TOP DEVICES
 common.bar.top-devices.description = Top devices
-common.bar.top-device-types = Top  Device types
+common.bar.top-device-types = TOP DEVICES TYPE
```

Comment:
it should stay as "Top Device types", please just remove the extra space in between "Top" and "Device".